frontend/fav_plants.html

```diff
@@ -0,0 +1,212 @@
+<!DOCTYPE html>
+<html lang="pt-br">
+
+<head>
+    <meta charset="utf-8">
+    <title>Comunidade de jardinagem</title>
+    <link rel="shortcut icon" href="images/lotus.png">
+    <link rel="stylesheet" href="index.css">
+    <link rel="stylesheet" href="fav_plants.css">
+</head>
+
+<body>
+    <div class="container">
+        <header id="cabecalho">
+            <button id="toggle-sidebar">☰</button>
+            <div class="logo">
+                <img src="images/lotus.png" alt="Ícone da comunidade de jardinagem">
+                <h1>Comunidade de jardinagem</h1>
+            </div>
+            <nav>
+                <span class="botao-header">
+                    <a href="">Página Inicial</a>
+                </span>
+                <span class="botao-header">
+                    <a href="">Perfil</a>
+                </span>
+                <span class="botao-header">
+                    <a href="">Plantas favoritas</a>
+                </span>
+            </nav>
+        </header>
+        <aside id="barra-lateral">
+            <p>Barra lateral</p>
+        </aside>
+        <main id="principal">
+            <section>
+                <h1 id="title">Comunidade de jardinagem</h1>
+                <button id="public-share">
+                    Compartilhe publicamente
+                </button>
+                <h2>Frutíferas</h2>
+            </section>
+            <section class="box-plantas">
+                <article class="post-plantas">
+                    <h3>Cajueiro</h3>
+                    <img src="images/lotus.png" class="foto-planta" alt="Icone pra ser substituído">
```

Comment:
Não vou ensinar o padre rezar a missa, vc já deve tá fazendo exatamente o que vou dizer...hehe, Eu não colocaria varias "articles". Penso que uma barra infinita ( que o usuario pode navegar  para os lados, e aparecer mais plantas ou voltar para a primeira seria interessante).
![image](https://user-images.githubusercontent.com/20483369/124046737-089c4b00-d9e9-11eb-892d-38c28c645f95.png)
Por exemplo:
Voce deixa 4 articles para cada tipo de planta, como vc fez, beleza?
Só que vc via js abastece essas 4 por imagens do banco, e ai o usuario clicando na seta da esquerda ou direita o js vai buscando mais fotos da categoria de forma lista circular.

Reply:
Boa ideia mano! Eu tinha pensado nisso de início. Vou deixar isso lá na issue de ToDo. Por hora, como eu mandei no comentário de cima, vamos tentar deixar só o layout básico nessa página, pra podermos acelerar e deixar todas as páginas existentes, depois a gente passe e refina tudo. Por hora, como conversamos no whatsapp, acho melhor deixar parecido com o _elevator pitch_ e o mais simples possível.